ui/tests/UHC-1.test.ts

```diff
@@ -0,0 +1,67 @@
+import { test, expect } from '@Test';
+import {generateRandomMail} from "./random";
+
+test.describe('UHC-1 (test ID)', () => {
+    test('Registration new customer with valid data and checking user data reset after logout', async ({ browser, page, baseURL  }) => {
+        await page.goto(baseURL as string, {waitUntil: 'domcontentloaded'});
+
+        const pageTitle = await page.title();
+
+        expect(pageTitle).toStrictEqual('UHC. United Healthcare.');
+
+        const myAccount = page.locator('//div[contains(@class, "myAccount__myAccountMenu")]');
+        await myAccount.click(); // клик (or focus?)
+
+        const accMenu = page.locator("//div[contains(@class, \"myAccount__myAccountMenu\")]/div[2]/div[contains(@class, \"topStripMenuDropdown__dropdownContent\")]");
```

Comment:
It's a very bad practice to build a spath by being tied to indexes
Indexes are unstable (layout can change during UI interaction) and are difficult to read